documentation/api/query/v4/aggregate-event-counts.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: "PuppetDB 3.2 » API » v4 » Querying Aggregate Event Counts"
+title: "PuppetDB 3.2: Aggregate event counts endpoint"
```

Comment:
Should we be removing the version? This seems relevant, later when we publish a v5 it might be important for users to distinguish and be clear what docs they are looking at.